tests/infra/consortium.py

```diff
@@ -118,11 +118,11 @@ def vote_using_majority(
         return res[1]
 
     def withdraw(self, member_id, remote_node, proposal_id):
-        return self._member_client_rpc_as_json(
-            member_id, remote_node, "withdraw", f"--proposal-id={proposal_id}"
-        )
+        with remote_node.member_client(member_id=member_id) as c:
+            return c.do("withdraw", {"id": proposal_id})
 
     def ack(self, member_id, remote_node):
+        # TODO: Produce signed ack here
```

Comment:
Doesn't the memberclient sign all commands since #611?

Reply:
The ack is not just a member-signed command - the body itself is a signature. We can do this in Python but haven't yet.